### PR TITLE
[api] [backend] Add support for k1om

### PR DESCRIPTION
**Pull request by seankelly**
Updated code with DB migration.

### CODE REVIEW

File: src/backend/BSConfig.pm.template [hunk 1]
```diff
@@ -122,6 +122,7 @@ our $relsync_pool = {
  "local" => "local",
  "i586" => "i586",
  "x86_64" => "i586",
+ "k1om" => "k1om",
```
Comment (adrianschroeter) [inline src/backend/BSConfig.pm.template]: not necessary, this is the default. Can you remove this part?

**Comment by adrianschroeter** [comment]
Can you please prefix the subject with our [api] and [backend] tags?

Apart from that it looks good :)

**Comment by bgeuken** [comment]
He meant the subjects of each commit, not of the pull request;-) Like here: 119908009192d

**Comment by seankelly** [comment]
With three commits, and likely another to fix Travis, should they all have the same prefix or should I tailor each commit to what it touches? E.g. the latest commit only touches src/backend/.. so would it get only [backend]?

**Comment by bgeuken** [comment]
Tailor it to what it touches.

The commits required to fix travis issues should idealy be rebased into the commit that caused the breakage.

**Comment by seankelly** [comment]
Someone else mentioned the 'k' in the migration's class name should be capitalized. Is this something that needs to be fixed?

**Comment by bgeuken** [comment]
Usually ruby class names should be written in CameCase. But in this case it doesn't really matter, since it's just a migration and a pretty short class name.

**Pull request merged**